floaters/src/main/java/com/mdg/droiders/floaters/ExpandedWindow.java

```diff
@@ -174,7 +184,7 @@ void toggleVisibiltyStatus(WindowManager wm) {
             WindowManager.LayoutParams lp = getWindowLayoutParams();
             setSheetHeight();
             setArrowPos();
-            lp.dimAmount = 0.4f;
+            lp.dimAmount = 0.7f;
```

Comment:
\[Checkstyle\] WARNING: 'if' child have incorrect indentation level 12, expected level should be 6\.